doc/data/datasets.rst

```diff
@@ -0,0 +1,27 @@
+Datasets
+========
+
+
+deep_qa.data.dataset
+--------------------
+
+.. automodule:: deep_qa.data.datasets.dataset
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Entailment
+----------
+
+.. automodule:: deep_qa.data.datasets.entailment.snli_dataset
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Language Modelling
```

Comment:
Isn't it "language modeling", with one l?  Oh, just looked it up, this is another US/UK difference.  I've been trying not to be picky about using American spellings, but I think this one deals with consistency in naming across modules.  We have `data.instances.language_modeling`, and `models.language_modeling`, but `data.datasets.language_modelling` - let's be consistent here.